src/components/TodoItem.tsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import classNames from 'classnames';
+import React from 'react';
+import { Todo } from '../types/Types';
+
+type TodoItemProps = {
+  todo: Todo;
+};
+
+export const TodoItem = ({ todo }: TodoItemProps) => {
```

Comment:
Let's extract values from todo here or at the line below to avoid typing `todo.` each time.